haskell/src/Monocle/Backend/Queries.hs

```diff
@@ -300,7 +300,7 @@ getQueryFromSL query =
 getSimpleTR :: BH.TermsResult -> TermResult
 getSimpleTR tr = TermResult (getTermKey tr) (BH.termsDocCount tr)
 
-data TermResult = TermResult {term :: Text, count :: Int} deriving (Show, Eq)
+data TermResult = TermResult {tRterm :: Text, tRcount :: Int} deriving (Show, Eq)
```

Comment:
We usually keep the prefix lowercase and make the actual record field PascalCase.
```suggestion
data TermResult = TermResult {trTerm :: Text, trCount :: Int} deriving (Show, Eq)
```

Reply:
Though, why not using the schema directly from SearchPB?

Reply:
If you don't mind I'll prefer to keep a segmentation here